src/AspNet.Security.OAuth.EVEOnline/EVEOnlineAuthenticationHandler.cs

```diff
@@ -23,33 +31,110 @@ public partial class EVEOnlineAuthenticationHandler : OAuthHandler<EVEOnlineAuth
         : base(options, logger, encoder, clock)
     {
```

Comment:
What happens with this constructor if there's no way to get a `JwtSecurityTokenHandler`? Will this additional constructor create DI ambiguity?